sklearn/feature_extraction/tests/test_text.py

```diff
@@ -857,6 +860,19 @@ def test_pickling_vectorizer():
         assert_array_equal(
             copy.fit_transform(JUNK_FOOD_DOCS).toarray(),
             orig.fit_transform(JUNK_FOOD_DOCS).toarray())
+ 
+    # Ensure that deleting the stop_words_ attribute doesn't affect transform
```

Comment:
Just make a separate test. This is no longer about pickling, and the current approach makes the above loop through `instances` more confusing.